kubernetes/updater/pkg/controller/podutils.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"github.com/gravitational/trace"
+	v1 "k8s.io/api/core/v1"
+)
+
+func getContainerImageFromPod(spec v1.PodSpec, container string) (string, error) {
```

Comment:
```suggestion
func getContainerImageFromPodSpec(spec v1.PodSpec, container string) (string, error) {
```

The same to `setContainerImageFromPod`